src/lib.rs

```diff
@@ -46,7 +46,7 @@ extern crate alloc;
 
 pub mod buf_encoder;
 pub mod display;
-mod error;
+pub mod error;
```

Comment:
Should this be `parse::error`? There are no non-parsing errors.

Reply:
`InvalidCharError` and `OddLengthStringError` are `iter` things.  We could group parse/iter separately to display/buf_encoder but for this little crate I don't think its worth it.

Reply:
Those are still parsing error but for some reason in the `iter` module. I think we should split the modules based on intended use not based on technical details.

Reply:
Bro, is it really necessary to hold up the release for this discussion - you are taking the piss right now.

Reply:
No, we can publish them later. I don't understand why we should do it right now.